robotbase/src/main/java/com/team2357/frc2022/commands/FailsafeCommand.java

```diff
@@ -0,0 +1,32 @@
+package com.team2357.frc2022.commands;
+
+import com.team2357.lib.commands.CommandLoggerBase;
+import com.team2357.lib.subsystems.ClosedLoopSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+public class FailsafeCommand extends CommandLoggerBase{
+    private boolean m_failsafeActive;
+    private ClosedLoopSubsystem m_subsystem;
+    
+    public FailsafeCommand(boolean failsafeActive, ClosedLoopSubsystem subsystem) {
+        m_failsafeActive = failsafeActive;
+        m_subsystem = subsystem;
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+
+        m_subsystem.setClosedLoopEnabled(!m_failsafeActive);
```

Comment:
I think this code can be a little clearer. I'd suggest something like the following:
```
if failsafe active
  setClosedLoopEnabled(false)
  cancel commands
else
  setClosedLoopEnabled(true)
```